<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Examen Departamental</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1E1E2E;
}

QWidget#centralwidget {
    background-color: #1E1E2E;
}

QPushButton#btnCalcular {
    background-color: #6C5CE7;
    color: white;
    border-radius: 20px;
    padding: 10px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton#btnCalcular:hover {
    background-color: #5B4BC4;
}

QLabel {
    font-size: 16px;
    font-weight: bold;
    padding: 10px;
    border-radius: 10px;
    margin: 5px;
}

QLabel#lblTitulo {
    color: #FFD700;
    font-size: 24px;
    font-weight: bold;
    background-color: #2D2D44;
}

QLabel#lblArea {
    color: #4CAF50;
    background-color: #233623;
    border: 1px solid #4CAF50;
}

QLabel#lblPython {
    color: #2196F3;
    background-color: #233246;
    border: 1px solid #2196F3;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitulo">
      <property name="text">
       <string>Examen Departamental</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCalcular">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: #2D2D44;
    border-radius: 15px;
    padding: 20px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblArea">
         <property name="text">
          <string>Área del Rectángulo: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPython">
         <property name="text">
          <string>Python Invertido: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_grafica">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: #2D2D44;
    border-radius: 15px;
    padding: 20px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
